src/com/trollworks/gcs/widgets/outline/MultiCell.java

```diff
@@ -205,10 +205,22 @@ public Cursor getCursor(MouseEvent event, Rectangle bounds, Row row, Column colu
         return Cursor.getDefaultCursor();
     }
 
+    /** Allow a satified row to display a tooltip */
```

Comment:
`satified` -> `satisfied`

Reply:
Is there a way to enable spell checking in comments?  I could certainly use it ;-)